package.json

```diff
@@ -5,22 +5,22 @@
   "main": "index.js",
   "scripts": {
     "test": "node --expose-wasm --harmony ./tests/allRunner.js",
-    "vmTests": "node --expose-wasm --harmony ./tests/runVmTests.js | tap-bail",
```

Comment:
Why remove this? it seems to be useful:
> Abort TAP test runners on the first failing assertion.

Reply:
It was causing the test runner to hang (not proceed to the next test) after some failures. This was the easy fix.